templates/_templating_scripting.py

```diff
@@ -149,11 +149,51 @@ def prompt_share(args: argparse.Namespace) -> None:
 
     This function is intended for running on a PR on a 'target repo'.
     """
+    pr_number = args.pr_number
+
     def gh_json(sub_command: str, field: str) -> dict:
         command = shlex.split(f"gh {sub_command} --json {field}")
         return json.loads(check_output(command))
 
-    pr_number = args.pr_number
+    # Get all commits in the PR.
+    commits = gh_json(f"pr view {pr_number}", "commits")["commits"]
+
+    # Get all authors found in the commits
+    all_authors = set(
+        commit_author
+        for commit in commits
+        for commit_author in commits
+    )
+    # Remove bots to get only *human* authors
+    human_authors = get_all_authors() - BOTS
```

Comment:
```suggestion
    human_authors = all_authors - BOTS
```

Reply:
As above, part of the previous mess, hopefully now fixed